packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -78,39 +79,56 @@ export const styles = theme => {
         top: 'initial',
         bottom: 0,
         width: 2,
+        height: '100%',
       },
     },
     /* Styles applied to the track element before the thumb. */
     trackBefore: {
       zIndex: 1,
       left: 0,
-      transition: trackTransitions,
+      transformOrigin: 'left bottom',
     },
     /* Styles applied to the track element after the thumb. */
     trackAfter: {
       right: 0,
       opacity: 0.24,
-      transition: trackTransitions,
+      transformOrigin: 'right top',
+      '&$vertical': {
+        top: 0,
+      },
+    },
+    /* Styles applied to the thumb wrapper element. */
+    thumbWrapper: {
+      position: 'relative',
+      zIndex: 2,
+      pointerEvents: 'none',
```

Comment:
@Pajn Just saw that this caused a regression with the hover state. This prevents `:hover` from being recognized on the button. Why do we need this property?

Reply:
Oops 😳 

I'm unsure. And removing it does not seem to cause any negative effects.
In an earlier version the wrapper did have as 2d size (now it's sized in the slider direction but zero in the other), with that I imagine that the click events may be placed wrong but now a hit test should never hit the wrapper anyway.

I would say that it is best to remove it as we can't explain why it's there, but if we want it, we can still fix the hover effect with `pointerEvents: 'auto'` on the thumb.